include/oneapi/dpl/pstl/hetero/dpcpp/utils_ranges_sycl.h

```diff
@@ -130,8 +150,13 @@ inline constexpr all_view_fn<sycl::access::mode::read, __dpl_sycl::__target_devi
 inline constexpr all_view_fn<sycl::access::mode::write, __dpl_sycl::__target_device, sycl::access::placeholder::true_t>
     all_write;
 
+#if _ONEDPL_SYCL_PLACEHOLDER_HOST_ACCESSOR_DEPRECATED
+//CPO to get a sycl::host_accessor instance due to non-placeholder sycl::accessor for the host is deprecated
```

Comment:
What do you think about removing that comment? `_ONEDPL_SYCL_PLACEHOLDER_HOST_ACCESSOR_DEPRECATED` macro name is pretty descriptive on its own.

Reply:
The comment part about CPO should probably be added prior to: 
```
    operator()(_R&& __r) const -> decltype(::std::forward<_R>(__r))
    {
        return ::std::forward<_R>(__r);
    }
```